_infra/helm/frontstage/values.yaml

```diff
@@ -96,4 +96,7 @@ gnu:
   fingerprint: ED1B7A3EADF95687
 
 password:
-  attemptTimeout: 86400
\ No newline at end of file
+  attemptTimeout: 86400
+
+frontstageProfiler:
```

Comment:
As per [this]https://github.com/ONSdigital/ras-rm-config/pull/236#pullrequestreview-1739976749 comment. The context of this is frontstage so no need to prefix it.

Reply:
Makes sense. This is updated.